assets/svelte/components/PageAstNode.svelte

```diff
@@ -45,6 +45,12 @@
     return !!htmlWrapper && htmlWrapper.getElementsByTagName("iframe").length > 0
   })()
 
+  $: {
+    if (isSelectedNode) {
+      setSelectedDom(domElement || htmlWrapper)
+    }
+  }
```

Comment:
This logic makes sure we set the right dom selection for the selection menu, in this case the delete button. I hope to use this same logic for an improvement on the drag-menu as well